usr/share/rear/layout/prepare/GNU/Linux/210_load_multipath.sh

```diff
@@ -1,7 +1,10 @@
 
-if grep -q '^multipath' "$LAYOUT_FILE" ; then
+# Activating multipath if BOOT_OVER_SAN variable is true
+# or if multipath device are present in LAYOUT_FILE
+if grep -q '^multipath' "$LAYOUT_FILE" || is_true "$BOOT_OVER_SAN" ; then
     Log "Activating multipath"
     modprobe dm-multipath >&2
+    [ ! -f /etc/multipath.conf ] && touch /etc/multipath.conf
```

Comment:
@schlomo this is on recovered system which is purely using SAN devices, therefore, a `/etc/multipath.conf` file must be created. For me that is correct.